packages/components/organisms/f-cookie-banner/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-cookie-banner",
   "description": "Fozzie Cookie Banner – Cookie Banner",
-  "version": "0.2.0",
+  "version": "0.2.1",
```

Comment:
As there's a fair bit of work in this PR maybe it should be a minor version bump?

Reply:
@DamianMullins Agreed